retrofit/WebServices/src/main/java/com/example/aperez/retrofittest/mvp/model/ResponseModel.java

```diff
@@ -47,4 +53,20 @@ public void onFailure(Call<LatestResponse> call, Throwable t) {
             }
         });
     }
+
+    public List<Image> getSavedImages() {
+        return Image.getAll();
+    }
+
+    public void saveImages(List<Image> images) {
+        ActiveAndroid.beginTransaction();
```

Comment:
As a good practice. You could create another class, to manage the Image table. For example, ImageRepository. Not necessary to apply change, only a sugerency